app-misc/splitvt/splitvt-1.6.6-r4.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+MY_P="${P/-/_}"
+DEB_PL="7"
```

Comment:
You might argue there's no point now as it looks gone from upstream, but for Debian patches, I normally version the ebuild like 'splitvt-1.6.6_p7.ebuild'.

Given we have to revbump for the fixes anyway and 1.6.6_p7 > 1.6.6-r* (it's not going to appear as a downgrade for users which is often confusing), let's rename.